cmd/cliui/main.go

```diff
@@ -37,6 +38,43 @@ func main() {
 		},
 	}
 
+	root.Children = append(root.Children, &serpent.Command{
+		Use: "colors",
+		Handler: func(inv *serpent.Invocation) error {
+			msg := pretty.Sprint(cliui.DefaultStyles.Code, "This is a code message")
+			_, _ = fmt.Fprintln(inv.Stdout, msg)
```

Comment:
You can simplify this with `pretty.Fprintf`

Reply:
Done! I had to separate the newline from the Fprintf call though as otherwise the newline would get themed.